Tools/autotest/arducopter.py

```diff
@@ -3832,86 +3832,6 @@ def SplineTerrain(self):
         self.set_parameter("TERRAIN_ENABLE", 0)
         self.fly_mission("wp.txt")
 
-    def WPNAV_SPEED(self):
```

Comment:
Also needs removing here: https://github.com/ArduPilot/ardupilot/blob/56773f09b422ece063b3531213e32618d04b1319/Tools/autotest/arducopter.py#L11740

Reply:
Also note that we have tests for `WPNAV_SPEED_UP` and `WPNAV_SPEED_DN`, do they need updating?